oap-server/server-library/library-datacarrier-queue/src/main/java/org/apache/skywalking/oap/server/library/datacarrier/consumer/MultipleChannelsConsumer.java

```diff
@@ -133,6 +133,28 @@ void shutdown() {
         running = false;
     }
 
+    void close(Channels channels) {
```

Comment:
If you are going to use this in a hot-update case, I have a concern whether this is thread safe. Because the reloading could happen in any time.